classify_resources_by_owners.py

```diff
@@ -0,0 +1,301 @@
+#!/usr/bin/env python
+# -- encoding: utf-8 --
+#
+# Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of FI-Core project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+#
+author = 'chema'
+
+import logging
+
+from osclients import osclients
+import openstackmap
+from settings.settings import TRIAL_ROLE_ID, COMMUNITY_ROLE_ID, BASIC_ROLE_ID,\
+    ADMIN_ROLE_ID
+import utils.log
+import sys
+import argparse
+
+
+class ClassifyResources(object):
+    """Class to analyse the resources by owner, for discovering resources
+       belonging to users that should not have resources, or resources without
+       an owner"""
+
+    def __init__(self, cache_dir, regions=None):
```

Comment:
Because regions is not a boolean, is a set. Anyway, I'll include in the docstring the description of the parameter.